golang/server/server.go

```diff
@@ -114,21 +117,45 @@ func (server MinimalGRPCServer) RunUntilStopped(stopper <-chan struct{}) error {
 	}
 
 	grpcServerResultChan := make(chan error)
```

Comment:
perhaps this could have a more generic name as everyone is piping into this including the mux